firestore/CONTRIBUTING.md

```diff
@@ -0,0 +1,82 @@
+# Overview
+
+This describes how Firestore Unity SDK works, and how to develop and test 
+the SDK, targeting desktop/Android/iOS.
+
+# Prerequisites
+
+Building the Unity SDK requires building the underlying C++ SDK. Refer to
+[][this doc] for what the prerequisites are.
+
+On top of above, you also need Unity installed (obviously). If you use an
+apple silicon machine as host, be sure to install the right version of
```

Comment:
nit: Explain what you mean by "right version", and what happens if you install the "wrong version".